app/assets/javascripts/viewers.js

```diff
@@ -0,0 +1,9 @@
+var onReadyViewers = function() {
+  if (newOrEdit(["moments", "strategies"])) {
+  	$("#viewers_all").click(function() {
+  		$("#viewers_list :checkbox").prop("checked", $(this).prop("checked"));
```

Comment:
My plan is once I merge this PR, I'll implement a similar auto-complete feature for Viewers so that instead of displaying a potentially long list of allies, users can just type in people.